docs/UserGuide.adoc

```diff
@@ -513,20 +513,20 @@ __Constraints:__
 // tag::list[]
 ==== Listing tasks : `list` +
 
-Want to go back to the default list of task after a search? Simply use the list command:
+When you want to go back to the default list of tasks after a search, you can use the list command to return to the default list of tasks.
 
 Format: `list`
 // end::list[]
 
 // tag::select[]
 ==== Selecting a task: `select`
+When you want to select a task, you can use the select command to select the task identified by the index number used in the last task listing.
```

Comment:
"select a task" has been repeated twice. You can either remove the "when you...a task" phrase or the "to select....task listing" phrase.